app/dts/behaviors/macros.dtsi

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2021 The ZMK Contributors
```

Comment:
Should all these copyright dates be 2022?

Reply:
Fixed.